### DISCUSSION TITLE
記事のページ上でユーザーをフォローしやすい仕組み

### DISCUSSION THREAD

**Discussion by yamadashy** [increments/qiita-discussions#96] [要望・アイデア・不具合]
（初投稿なので何か間違いがあればご指摘いただけますと 🙇🏻  ）

# 概要
DEV や Zenn のように「記事を読みつつすぐにフォローできる仕組み」があると、
より気軽にユーザーをフォローできて、コミュニティが活性化するのではないかと思いました。

現状の Qiita だと記事の最下部かユーザーのページに行く必要があり、この遷移の面倒さが障壁となっているように感じます。

何か理由があって設置していないのであれば、その理由をご教授いただけますと幸いです。

### DEV
画面右側に、ユーザーの概要と一緒に表示されております。
<img width="1275" alt="スクリーンショット 2021-12-26 17 14 15" src="https://user-images.githubusercontent.com/5019072/147402616-fe8656a9-1321-461f-9d8f-d90ea41ffed0.png">

### Zenn
DEV と同様です
<img width="1363" alt="スクリーンショット 2021-12-26 17 14 50" src="https://user-images.githubusercontent.com/5019072/147402627-64a5f0b4-c12a-40db-bff8-581c701d2f54.png">

# メリット
気軽にフォローできることによる効果
- タイムラインが充実しやすくなり、より多くの記事に目を通すきっかけになる
- フォローされたユーザーのモチベーションにつながる。記事も読まれやすくなる

UI変更による副次的効果
- 筆者の概要が前面に表示されることで、
    - プロフィールが読者の目に触れる機会が増えるため、記事を書く価値が高まる
    - どのようなユーザーがその記事を書いているかがわかり、記事への信頼感などに繋がる？ LGTMも押されやすくなる？

# デメリット
- 要素が増えることにより UI がわかりづらくなる可能性がある
- そのユーザーのことをきちんと確認せずにフォローできるため、ユーザーのタイムラインが意図しないものになる可能性はある

# その他
Qiita には Organization もあるので、そちらの情報も一緒に表示・フォローできるようにするのは悩ましいところです

**Comment by kskwtnk** [comment]
@yamadashy 
ご意見ありがとうございます！

こちらは非常に実施したい施策です！
ただ、Qiitaの場合は広告があり、収益として外せない＆位置を下げたりもできず…… 🙏 

そのため、プロフィールやフォロー用UIをどこに置こうか非常に悩ましく、すぐの実施が難しそうです 🙇 
Issueを立てるなど、具体的なアクションができず申し訳ありませんが一旦現状をご理解いただけますと幸いです 🙇

**Comment by yamadashy** [comment]
ご返信ありがとうございます 🙇🏻  
（確認漏れておりました。申し訳ありません）

確かに現状だと厳しそうですね...。

スクロールすると広告が一つになって空間ができるのでそこに..と思いましたが、それはそれで目次も下がり見た目もごちゃっとしそうですね。

丁寧なご回答ありがとうございました。
クローズしていただけますと 🙇🏻

**Reply by kskwtnk** [reply]
@yamadashy 

いえいえ！お返事いただきありがとうございます！

引き続き検討中ではありますので、近いうちに実現できるように頑張ります！
